kbfsgit/runner.go

```diff
@@ -251,8 +251,14 @@ func (r *runner) initRepoIfNeeded(ctx context.Context, forCmd string) (
 		}()
 	}
 
-	fs, _, err = libgit.GetOrCreateRepoAndID(
-		ctx, r.config, r.h, r.repo, r.uniqID)
+	// Only allow lazy creates for public and multi-user TLFs.
```

Comment:
Is that because we don't provide a UI for these?

Reply:
Yeah, there would be no other way to create public or mutli-user repos otherwise.